### PR TITLE
[Feature : 2D Map] Colour tour graph based on power values

### PR DESCRIPTION
**Pull request by cyc1ingsir**
Related Feature Request: https://github.com/mytourbook/mytourbook/issues/1071
I hoped this would be a good first (or second, to be precise) time issue for a PR.
So I had a go.
Unfortunately, my setup is completely broken, so I'm not able to run MT within Eclipse any longer, nor is `mvn  install` working for me.

I'm still hoping, I discovered every place within the code base, where an addition is needed.
May I kindly ask for a review and guidance on how I may improve this code.

### CODE REVIEW

**Review by wolfgang-ch — CHANGES_REQUESTED** [review]
For your "first" PR there are a few issues.

I found no issues for the code itself, but [here](https://github.com/mytourbook/mytourbook/discussions/919) are a few rules which I update infrequently.

**Comment by cyc1ingsir** [comment]
I hope to have covered everything by the last commits.
Eclipse keeps to rearranging some code in some of the changed files on me and being used to use a completely different IDE (at work), I couldn't find a way to preventing it doing so, swiftly. :-(
So I just hope, the commits include all the necessary changes but also not more changes than needed.

**Comment by wolfgang-ch** [comment]
> Eclipse keeps to rearranging some code in some of the changed files on me and being used to use a completely different IDE (at work), I couldn't find a way to preventing it from doing so, swiftly. :-(

Have you setup the code formatting as described in https://mytourbook.sourceforge.io/mytourbook/index.php/development/dev-code-style/code-style

**Comment by wolfgang-ch** [comment]
A very colorful track when power values are computed and not from a device.

The power max value of 800 Watt which you have set, was a little bit too high for me

image

File: bundles/net.tourbook/src/net/tourbook/map2/view/SlideoutMap2_TourColors.java [hunk 1]
```diff
@@ -55,6 +55,7 @@ public class SlideoutMap2_TourColors extends ToolbarSlideout implements IActionR
    private Button   _chkShowInChartToolbar_Altitude;
    private Button   _chkShowInChartToolbar_Gradient;
    private Button   _chkShowInChartToolbar_Pace;
+   private Button   _chkShowInChartToolbar_Power;
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/map2/view/SlideoutMap2_TourColors.java:58]: This comment is for the following line in this file

` * Copyright (C) 2005, 2021 Wolfgang Schramm and Contributors`

When you modify a file you may also check, that the copyright year is updated. This is not dramatically and I also forget it sometimes.

**Review by wolfgang-ch — APPROVED** [review]
There is no need for further modifications

**Pull request merged**